reference-implementation/lib/transform-stream.js

```diff
@@ -259,30 +259,44 @@ function SetUpTransformStreamDefaultController(stream, transformAlgorithm, flush
 }
 
 function SetUpTransformStreamDefaultControllerFromTransformer(stream, transformer) {
-  function transformAlgorithm(chunk) {
```

Comment:
That got a lot simpler, yay.

Reply:
Not as simple as I'd like, but all the other algorithm creation steps are looking great now.